docs/configuration/validation/brk.rst

```diff
@@ -0,0 +1,43 @@
+.. _configuration_validation_brk:
+
+=========================
+BRK - Zakelijk gerechtigd
+=========================
+
+The special *AddressNL* component has a specific validator available, that can
+check if the currently logged in user is the business owner of the filled-in property.
+
+Configuration
+=============
+
+1. Obtain a BRK API key.
+2. In Open Forms, navigate to: **Miscellaneous** > **KvK configuration**.
+3. Click **Add** next to the **KvK API Basisprofiel** field and fill in the following
+   details:
```

Comment:
I think we preferably have people add the service via **Configuration > Services** and then make them select the option in the dropdown rather than adding it through the popup/window.